.github/workflows/ci_test-full.yml

```diff
@@ -40,8 +40,8 @@ jobs:
       with:
         python-version: ${{ matrix.python-version }}
 
-    - name: Weekly reset caching
-      run: echo "::set-output name=period::$(python -c 'import time ; days = time.time() / 60 / 60 / 24 ; print(int(days / 7))' 2>&1)"
+    - name: Reset caching
+      run: echo "::set-output name=period::$(python -c 'import time ; days = time.time() / 60 / 60 / 24 ; print(f"d{int(days / 2) * 2}")' 2>&1)"
```

Comment:
Why the added `d`?
And why the `* 2`?

Reply:
https://github.com/PyTorchLightning/pytorch-lightning/pull/12405#discussion_r831718747